src/StrippetsVisual.ts

```diff
@@ -575,6 +588,25 @@ export default class StrippetBrowser16424341054522 implements IVisual {
         return outlinesInstance;
     }
 
+    // https://stackoverflow.com/questions/35962586/javascript-remove-inline-event-handlers-attributes-of-a-node#35962814
+    public static removeScriptAttributes(el) {
+        const listOfEvents = [];
```

Comment:
is listOfEvents being used in anywhere? If not, I think it's better to remove it for simpler code.